src/screens/bugReportScreen/index.js

```diff
@@ -1,12 +1,42 @@
 import React, { useState, useEffect } from 'react';
 import { getBugReports } from '../../firebase/manage';
+import { Label } from 'semantic-ui-react';
+
+const styles = {
+  header: {
+    color: 'black',
+    backgroundColor: 'lightBlue',
+    marginBottom: '5px',
+    marginTop: '20px',
```

Comment:
Can we replace any filed values, that is pixels, with percentages or variable styling like 'flex'? That will make it work consistently across various screen sizes.